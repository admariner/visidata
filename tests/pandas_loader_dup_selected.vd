sheet	col	row	longname	input	keystrokes	comment
global		filetype	set-option	pandas		
			open-file	sample_data/benchmark.csv	o	
benchmark		0	select-row		s	select current row
benchmark		1	select-row		s	select current row
benchmark		2	select-row		s	select current row
benchmark			dup-selected		"	open duplicate sheet with only selected rows
