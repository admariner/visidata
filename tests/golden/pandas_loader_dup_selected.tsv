Date	Customer	SKU	Item	Quantity	Unit	Paid
7/3/2018 1:47p	Robert Armstrong	FOOD213	BFF Oh My Gravy! Beef & Salmon 2.8oz	4	$12.95	$51.8
7/3/2018 3:32p	Kyle Kennedy	FOOD121	Food, Adult Cat - 3.5 oz	1	$4.22	$4.22
7/5/2018 4:15p	Douglas "Dougie" Powers	FOOD121	Food, Adult Cat 3.5 oz	1	$4.22	$4.22
